src/static/css/misc.less

```diff
@@ -1155,3 +1155,35 @@
     letter-spacing: 1px;
     text-transform: uppercase;
 }
+
+
+/* topdoc
```

Comment:
YES! Thanks! Fixed in https://github.com/cfpb/cfgov-refresh/commit/734399a357b5c8b0c63bdf399ffc441a0aab48c9